### DISCUSSION TITLE
Does mkdocs have a method to sponsor the project?

### DISCUSSION THREAD

**Discussion by josegonzalez** [mkdocs/mkdocs#2969] [Q&A] [answered]
I was wondering this based on the fact that the Material for Mkdocs (which I use in Dokku) does. While considering sponsoring _that_ project, I was wondering about this one, which is a large reason for the existing of Material for Mkdocs.

Does this project have any ideas around doing sponsorship in some way?

**Comment by oprypin** [comment]
Unfortunately, MkDocs does not accept donations or provide paid support.
If that ever changes, I will reply here again.
Please react with "👀" if you're interested in updates on that.

**Reply by squidfunk** [reply]
I already offered to push some of the funds Material for MkDocs receives upstream to MkDocs, but as @oprypin said, this project doesn't accept funds (yet?). From what I know, at least some maintainers are not allowed to accept funds due to the contracts they have with their employers. On the contrary, I'm an independent engineer, which is why this is not a problem for Material for MkDocs.

If that changes, I'm happy to help with funding.

**Reply by josegonzalez** [reply]
Not an admonition at all of your work @squidfunk, just thought I'd spread the wealth :)

**Reply by squidfunk** [reply]
No worries, and thanks for considering sponsoring!

**Reply by ultrabug** [reply]
This discussion made me realize that it seems that I am the only one from the [past and current maintenance team](https://github.com/mkdocs/mkdocs/blob/master/docs/about/release-notes.md#maintenance-team) of MkDocs who is able to receive donations :(

[EDIT]
~I'd be happy to discuss this and help if the others have an interest if proposing a fair way to set this up for the MkDocs project.~

Since -as Martin mentioned- some active maintainers are prevented from accepting funds, there is currently no way to handle this in a fair manner. If that changes, happy to help too!

**Reply by oprypin** [reply]
As of now ([announced with the 1.5.0 release](https://github.com/mkdocs/mkdocs/releases/tag/1.5.0)) you can show your support for MkDocs through GitHub Sponsors. This can be found on the right side panel of [the repository](https://github.com/mkdocs/mkdocs). One-time or recurring donations of any amount are welcome.

According to the ":eyes:" reactions, tagging people: @pawamoy @squidfunk @josegonzalez @mondeja @ultrabug @wilhelmer @nejch @drunkwcodes

**Comment by oprypin** [comment]
Hi!
(Reaching out primarily to past maintainers - @tomchristie, @d0ugal, @waylan. Will also try to find your emails and direct you here.)

I am considering opening up donations on my GitHub profile and then directing people just to my profile if they wish to send donations for MkDocs.

I understand that this would leave past maintainers and current contributors without any compensation. But then again, for the past 2 years it has been just me setting the direction for the project. And I am only now getting somewhat comfortable touching the topic of donations.

This option is attractive to me also because otherwise we'd need to set up actual accounting. Instead let's keep the donations personal; we can additionally list the current active contributors so that if people wish to sponsor them they get pointed to their direction as well.

And do I understand correctly that MkDocs never accepted funds even historically? Are past maintainers interested in splitting this instead? (As for current contributors, I have already expressed my intention to them a bit earlier.) Or if there are overall concerns about changing the status after such a long history, we could also freeze the idea.

**Reply by pawamoy** [reply]
Glad to see you'll accept donations :slightly_smiling_face: I just started sponsoring @ultrabug (yesterday! should have started before), happy to start sponsoring you too once your account allows it :slightly_smiling_face: (and any other current/past maintainer if they accept donations, particularly @waylan who is maintaining Python-Markdown).

**Reply by waylan** [reply]
For the record I have no interest in receiving any donations personally and am indifferent to what choice MkDocs makes about this going forward.

**Reply by lovelydinosaur** [reply]
@oprypin - Good for you.👍

> This option is attractive to me also because otherwise we'd need to set up actual accounting.

I don't think you'd find any practical difference between having the funding be on the org or on your personal site. For myself I'd probably find sponsoring on `mkdocs` more compelling. You're more than welcome to handle it however you'd like, tho. 😊

**Reply by d0ugal** [reply]
Agreed with the above. I would have been interested long ago, but I think when I was last actively working on MkDocs then GitHub sponsors was not a thing 😀

**Comment by oprypin** [comment, marked as answer]
As of now ([announced with the 1.5.0 release](https://github.com/mkdocs/mkdocs/releases/tag/1.5.0)) you can show your support for MkDocs through GitHub Sponsors.
This can be found on the right side panel of [the repository](https://github.com/mkdocs/mkdocs).
One-time or recurring donations of any amount are welcome.
